packages/sdk/src/client.ts

```diff
@@ -1877,6 +1882,53 @@ export class Client
         )
     }
 
+    // upload transactions made on the base chain
+    async addTransaction(
+        chainId: number,
+        receipt: ContractReceipt,
```

Comment:
I wonder if we can keep ethers out of the sdk? Or is it in already elsewhere? I still have hope and plans that we can refactor to viem or upgrade. Based on what's passed to the stream, maybe custom type

```
transactionHash: string`
to: string
from: string
logs: {
  address: string
  data: string
  topics: string[]
}[]
```

Reply:
it is all over, but we can change this!